examples/text/document_clustering.py

```diff
@@ -102,7 +102,13 @@
 print(__doc__)
 op.print_help()
 
-(opts, args) = op.parse_args()
+
+def is_interactive():
+    import __main__ as main
+    return not hasattr(main, '__file__')
+
+argv = [] if is_interactive() else sys.argv[1:]
+(opts, args) = op.parse_args(argv)
 if len(args) > 0:
```

Comment:
Sorry, I can't follow your question completely. Like, we need to check for the `args` being empty and we use `opts` later in the code. Is there anything else you were looking for. Please correct me if I misunderstood you.